components/Carousel/Carousel.js

```diff
@@ -1,8 +1,48 @@
 class Carousel {
+    constructor(photoCarousel) {
+        this.photoCarousel = photoCarousel;
 
+        this.leftButton = document.querySelector('.left-button');
+        this.rightButton = document.querySelector('right-button');
+        this.photoData = this.photoCarousel.dataset.tab;
+
+        this.photo = document.querySelectorAll(`.image[data-tab='${this.photoData}']`);
+
+        this.photo = new Image(this.imageContent);
+
+        this.leftButton.addEventListener('click', () => {this.leftClick()});
+        this.rightButton.addEventListener('click', () => {this.rightClick()});
+
+    };
+
+    leftClick() {
+        let pictures = document.querySelectorAll('.image');
+        pictures.forEach(picture => picture.classList.remove('img'));
+        this.photoCarousel.classList.add('active-img');
+        this.pictures.forEach(pictures => (pictures.selectImage));
+    }
+
+    rightClick() {
+        let pictures = document.querySelectorAll('.image');
+        Array.from(pictures).forEach(picture => picture.classList.remove('img'));
+        this.photoCarousel.classList.add('active-img');
+        this.pictures.forEach(pictures => (pictures.selectImage));
+    }
+}
+
+
+class Image {
+    constructor(imageContent) {
+        this.imageContent = imageContent;
+    }
+
+    selectImage() {
+        this.imageContent.style.display = 'flex';
+    }
```

Comment:
Interesting way of handling the carousel :thinking: I would be very interested in seeing this completed.